cmd/crio/main.go

```diff
@@ -269,6 +269,18 @@ func main() {
 			return err
 		}
 
+		// Print the current cli flags
+		flagNames := c.FlagNames()
+		logrus.Infof("Cli Flags used:")
+		for _, flagName := range flagNames {
+			flagvalues := c.Value(flagName)
```

Comment:
Go's style is `camelCase` for variable names.

```suggestion
			flagValues := c.Value(flagName)
```